portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -921,6 +922,502 @@ definition {
 			userScreenName = "ldapusercn");
 	}
 
+	@description = "This is a use case for LPS-105170 TC-2: User creation and sync after SAML setup, using email Address for IdP Identifier and screenName for Attribute matching"
+	@priority = "3"
+	test CreateAndSyncUserAfterSAMLSetupMatchingemailAddressNameIDWithscreenNameAttribute {
+		property custom.properties = "jsonws.web.service.paths.excludes=";
+		property portal.upstream = "true";
+		property using.virtualInstances = "true";
+
+		// Enable SAML debug logs
+
+		SAML.enableSAMLDebugLogs();
+
+		// Add new virtual instance to be the SP site
+
+		SAML.addVirtualInstances(virtualHosts = "www.able.com");
+
+		User.logoutPG();
+
+		// Setup localhost as IDP role
+
+		SAML.setupIDP(
+			idpURL = "http://localhost:8080",
+			samlEntityID = "samlidp",
+			userEmailAddress = "test@liferay.com");
+
+		SAML.setupSP(
+			encryptionCertificate = "true",
+			samlEntityID = "samlsp",
+			spURL = "http://www.able.com:8080",
+			userEmailAddress = "test@www.able.com");
+
+		SAML.addIDPConfigurationsToServiceProvider(
+			idpAttributeMapping = "screenName,emailAddress,firstName,lastName,uuid",
+			idpEntityID = "samlidp",
+			idpKeepAliveURL = "http://localhost:8080/c/portal/saml/keep_alive",
+			idpURL = "http://localhost:8080/",
+			index = "0",
```

Comment:
As I remember, this needs some clever naming to reflect the business logic we set in our tests.